src/EloquentHashids.php

```diff
@@ -51,4 +51,12 @@ public static function getHashidEncodingValue(Model $model)
   {
     return $model->getKey();
   }
+  
+  /**
+   * @return string
+   */
+  public function getRouteKeyName()
+  {
+      return 'uid';
```

Comment:
Surely this should return the result of the hashid column? That way, it only needs to be set once.